pset-functions.js

```diff
@@ -208,6 +258,23 @@ console.log('Problem 5 - bonus:')
 
 // Add your code below this line
 
+function wordCount(phrase) {
+  let words = phrase.replace([","], "").split(" ")
+  let phraseCount = {}
+
+  words.forEach((word) => {
```

Comment:
Brilliant use of a forEach and if/else to fill an empty object with a key/value pair; and keep count! Lots of great stuff happening here.

You really knocked this homework out of the park! Keep up the great work.